pkg/controllers/raycluster_webhook_test.go

```diff
@@ -230,11 +233,23 @@ func TestRayClusterWebhookDefault(t *testing.T) {
 
 func TestValidateCreate(t *testing.T) {
 	test := support.NewTest(t)
-
+	emptyRayCluster := &rayv1.RayCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      rayClusterName,
+			Namespace: namespace,
+			Annotations: map[string]string{
+				versionAnnotation: "0.0.0",
+			},
+		},
+		Spec: rayv1.RayClusterSpec{},
+	}
 	validRayCluster := &rayv1.RayCluster{
```

Comment:
_no action comment_
Using annotation and functions for `validRayCluster` is correct as those values are set by default webhook.